usaspending_api/common/helpers/spark_helpers.py

```diff
@@ -155,6 +161,17 @@ def configure_spark_session(
         # conf.set("spark.hadoop.fs.s3a.committer.staging.conflict-mode", "fail")
         # conf.set("spark.hadoop.fs.s3a.committer.staging.tmp.path", "tmp/staging")
 
+        # Turn on Hive support to use a Derby filesystem DB as the metastore DB for tracking of schemas and tables
+        enable_hive_support = True
```

Comment:
So when `CONFIG.USE_AWS` is set to `True`, the input parameter value is ignored and `enable_hive_support` is set to `True`?

Reply:
Other way around. 

If USE_AWS is true, it will fall back to what the input parameter is. 

If USE_AWS is false/None, (e.g. when local), it will _always_ override that param and enable hive support. 

This may not be super intuitive that someone could pass in False, and it still be True due to it running locally. Thinking about if there's a better pattern.